teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] The `parent_permission` uses colon notation, so its application as an argument in the `#super` method should be slightly different.

  ```
  super(age, name, parent_permission: parent_permission)
  ```